packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -0,0 +1,50 @@
+import React, { Fragment } from 'react';
+import { shape, string } from 'prop-types';
+
+import { Link } from '@magento/venia-drivers';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import defaultClasses from './accountMenuItems.css';
+
+const MENU_ITEMS = [
```

Comment:
Would this be driven by admin settings in the future?

Reply:
That's a good thought, it certainly could be. I separated it out into this "configuration" array with localization in mind primarily.